Automattic/wp-calypso
# Social Signup: Add email button options

Related to #83872
Fixes https://github.com/Automattic/wp-calypso/issues/83872

## Proposed Changes

* Add email quick links when magic link is sent to users, more info https://github.com/Automattic/wp-calypso/issues/83872

<img width="1117" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/c1c69581-7fd8-4273-8aef-6b6f92143008">

<img width="1109" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/07cddb76-6b6b-407f-9b7f-3b89f23dc0e7">


## Testing Instructions



* Pull and run this branch OR use live link
* Navigate on incognito to `/log-in/link` and use  a “fake email” like 'blabla@fake.com to see all the buttons “shortcuts” available:
 - Gmail
 - AOL
 - yahoo
 - Outlook
 - Apple

If users' email is one of the ones mentioned above, we just show that button, if it is unknown, we show all 5.


C/c @crisbusquets, in the designs I noticed that the bottom part (Stay logged in with the Jetpack Mobile App)of the page is gone, I haven't removed it, should I? 🤔 

## Pre-merge Checklist
